qutebrowser/browser/commands.py

```diff
@@ -1231,6 +1250,12 @@ def _run_userscript(self, cmd, *args, verbose=False):
             'QUTE_MODE': 'command',
         }
 
+        try:
+            # not sure if int(winId()) is safe to call on every os.
+            env['QUTE_WID'] = str(int(self._tabbed_browser.winId()))
```

Comment:
`CommandDispatcher` has a `_win_id` attribute.

Reply:
Not the same thing though - that's qutebrowser's window ID (which is just a counter starting with 0), while here, we want the native OS window ID (to e.g. make mpv draw over it).

Reply:
Ah, got it

Reply:
I wonder if this should be called `QUTE_NATIVE_WID` or so to make it clearer.

Reply:
I think it's also interesting to mention that qutebrowser creates a window per tab, so third party programs could simply draw above a tab and therefore a user could switch between e.g. watch a video and doing something else in another tab.
The window id of the currently focused tab is stored in:
```python
int(self._tabbed_browser._now_focused.winId())
```